Phonebook.Frontend/src/app/pages/page-information/page-information.component.html

```diff
@@ -8,9 +8,9 @@ <h1 i18n="PageInformationComponent|Title of the page information@@PageInformatio
       Features are still subject to change without further notice and your data may get lost.
     </p>
   </div>
-  <div class="pb-flex pb-center-element">
+  <div class="pb-flex cardcontainer">
```

Comment:
fix this...

Reply:
What should be fixed?

Reply:
Ups sorry.. mistake comment to show something :D